test/integration/container/multimaster_test.go

```diff
@@ -270,6 +274,32 @@ data:
     baseurl=http://%s
     enabled=1
     gpgcheck=0
+
+  cloud-google-com.gpg.b64: |
```

Comment:
This change surely belongs together with the similar change to an example in the other commit.
Though I still can't see where it gets used.

Reply:
This key is referenced [here](https://github.com/weaveworks/wksctl/blob/master/pkg/plan/recipe/install_plans.go#L185) but was missing from the test files.